apertium-tur-uzb.tur-uzb.lrx

```diff
@@ -1,18 +1,119 @@
 <rules>
 
-<rule weight="1"> 
-    <match lemma="boyunca" tags="post">
-    <select lemma="bo'ylab" tags="post"/>
-    </match>
-</rule>
-
-<rule weight="2"> 
-    <match lemma="*" tags="num"/>
-    <match lemma="*" tags="n.*"/>
-    <match lemma="boyunca" tags="post*">
+  <rule weight ="1">  <!-- default -->
+  <match lemma="dışarı">
+    <select lemma="tashqari"/>
```

Comment:
where is tag?

Reply:
Same as above

Reply:
You should match lemmas and tags at the same time.

Reply:
Done, added tag too.